.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x android/gradlew
     - name: Build with Library with Gradle and check lints
-      run: cd android;./gradlew build test goJF
+      run: cd android;./gradlew build test verGJF
```

Comment:
https://github.com/sherter/google-java-format-gradle-plugin

I was confused about this. Now I understood!